src/units.cpp

```diff
@@ -206,6 +206,11 @@ bool updateUnitMultiplier(double &multiplier,
 {
     bool updated = false;
     auto unitsName = units->name();
+
+    if (units->isImport()) {
```

Comment:
I don't think we should do this. The error is in `isBaseUnit()` which should be covered in #478, so I'd suggest removing this check here.

Reply:
So for now, simply remove the check and address it in a further pull request which focuses on #478? It will only change the one unit test.